Dockerfile

```diff
@@ -14,7 +14,7 @@ WORKDIR /go/src/github.com/minio/
 COPY dockerscripts/docker-entrypoint.sh dockerscripts/healthcheck.sh /usr/bin/
 
 RUN  \
-     apk add --no-cache ca-certificates && \
+     apk add --no-cache ca-certificates curl && \
```

Comment:
While we're at it, can we update Dockerfile to pick up the curl version 7.61.0-r0 or above as @sanvarm2 suggested https://github.com/minio/minio/issues/6343#issue-353237646

Reply:
It picks up automatically the fixed version @nitisht we don't need to specifically stick any version. But we can perhaps add > 7.61 deps